scripts/warnings-handling.py

```diff
@@ -0,0 +1,31 @@
+import os
+import re
+import sys
+
+
+warning_matcher = re.compile(r'.*warning[: ].*', re.I | re.S)
+warnings = 0
+
+def warnings_count():
+    global warnings
+    with open(os.path.join(os.getcwd(), 'build-log.txt'), 'r') as build_log:
+        build_log_lines = build_log.readlines()
```

Comment:
It make sense to check regex on the go, but not store log on RAM. It's more efficient solution and makes code shorter.

Reply:
Changed the logic to read lines using a loop.